report/id_list.go

```diff
@@ -1,19 +1,25 @@
 package report
 
-import (
-	"sort"
-)
+import "sort"
 
-// IDList is a list of string IDs, which are always sorted and
-// without duplicates.
+// IDList is a list of string IDs, which are always sorted and unique.
 type IDList []string
 
-// NewIDList makes a new IDList.
-func NewIDList(ids ...string) IDList {
+// MakeIDList makes a new IDList.
+func MakeIDList(ids ...string) IDList {
 	sort.Strings(ids)
 	return IDList(ids)
 }
 
+// Copy returns a value copy, useful for tests.
+func (a IDList) Copy() IDList {
```

Comment:
It's primarily for tests, yes. But I can't move it there, because that's package report_test, and you can't define methods on types outside of their package. I could convert it to a free function. Is your concern simply pollution of the exported symbols?